frontend/src/queries/nodes/DataNode/Reload.tsx

```diff
@@ -0,0 +1,21 @@
+import { useActions, useValues } from 'kea'
+import { dataNodeLogic } from '~/queries/nodes/DataNode/dataNodeLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconRefresh } from 'lib/components/icons'
+import { Spinner } from 'lib/components/Spinner/Spinner'
+
+export function Reload(): JSX.Element {
+    const { responseLoading } = useValues(dataNodeLogic)
+    const { loadData } = useActions(dataNodeLogic)
+
+    return (
+        <LemonButton
+            type="secondary"
+            onClick={loadData}
+            disabled={responseLoading}
+            icon={responseLoading ? <Spinner /> : <IconRefresh />}
```

Comment:
I'd use LemonButton's loading prop for displaying the spinner here again, i.e. suggest to replace the above two lines with:
```
loading={responseLoading}
icon={<IconRefresh />}
```